{
    "config": {
        "error": {
            "cannot_connect": "Falha ao conectar",
            "unknown": "Erro inesperado"
        },
        "step": {
            "user": {
                "data": {
                    "latitude": "Latitude",
                    "longitude": "Longitude"
                }
            }
        }
    }
}